blog/docs/articles/workflow-as-function-flow.md

```diff
@@ -0,0 +1,328 @@
+
+````
+mvn clean package -Pcontainer-native
+````
+
+This command builds a container with the service that is using native executable built with GraalVM. The build process might take a while.
+
+### Push built container to registry
+
+After the container image is built, push it to an external registry that your Knative cluster can pull from.
+
+### Deploy to Knative cluster
+
+1. Create a Knative eventing broker, for example by using following command:
+
+	```
+	kubectl apply -f - << EOF
+	apiVersion: eventing.knative.dev/v1
+	kind: broker
+	metadata:
+	 name: default
+	 namespace: knative-eventing
+	EOF
+	```
+
+	Complete deployment file is generated as part of the build and can be found in `target/functions/user-registration-{version}.yaml`. To deploy it issue following command:
+
+	```
+	kubectl apply -f target/functions/user-registration-{version}.yaml
+	```
+
+	This will provision complete service and all the Knative Eventing triggers. In addition it will also create sink binding to make the service an event source to be able to publish events from function execution.
+
+2. Optionally deploy `cloudevents-player` that will display all events flowing through the broker with following commands
+
+
+	```
+	kubectl apply -n knative-eventing -f - << EOF
+	apiVersion: serving.knative.dev/v1
+	kind: Service
+	metadata:
+	  name: cloudevents-player
+	spec:
+	  template:
+	    metadata:
+	      annotations:
+	        autoscaling.knative.dev/minScale: "1"
+	    spec:
+	      containers:
+	        - image: quay.io/ruben/cloudevents-player:latest
+	          env:
+	            - name: PLAYER_MODE
+	              value: KNATIVE
+	            - name: PLAYER_BROKER
+	              value: default
+	---
+	apiVersion: eventing.knative.dev/v1
+	kind: Trigger
+	metadata:
+	  name: cloudevents-player
+	  annotations:
+	    knative-eventing-injection: enabled
+	spec:
+	  broker: default
+	  subscriber:
+	    ref:
+	      apiVersion: serving.knative.dev/v1
+	      kind: Service
+	      name: cloudevents-player
+	EOF
+	```
+
+3. Get the url of the default broker use following command
+
+	```
+	kubectl get broker default
+	```
+
+4. Send request to the broker to start user registration
+
+	```
+	curl -v "http://broker-ingress.knative-eventing.svc.cluster.local/knative-eventing/default" \
+	-X POST \
+	-H "Ce-Id: 1234" \
+	-H "Ce-Specversion: 1.0" \
+	-H "Ce-Type: io.automatiko.examples.userRegistration" \
+	-H "Ce-Source: curl" \
+	-H "Content-Type: application/json" \
+	-d '{"user" : {"email" : "mike.strong@email.com",  "firstName" : "mike",  "lastName" : "strong"}}'
+	```
+
+	This will go via number of events being exchanged over the Knative broker and invoking functions defined in the workflow. It also uses Swagger Petstore REST api so in case of successful user registration it will be visible in Swagger Petstore as new user.
+
+	**NOTE** That Swagger Petstore does not have reliable storage thus it might require few get requests to be issued to see it there.
```

Comment:
```suggestion
	This will go by number of events being exchanged over the Knative broker and invoking functions defined in the workflow. It also uses Swagger Petstore REST API, so in case of successful user registration it will be visible in Swagger Petstore as new user.

	!!! note
	    That Swagger Petstore does not have reliable storage thus it might require few get requests to be issued to see it there.
```

Reply:
please check out https://knative.dev/docs/help/contributor/mkdocs-contributor-guide/#admonitions for info about notes etc